### PR TITLE
HARMONY-1671: Use https for load balancers

### PR DESCRIPTION
**Pull request by chris-durbin**
## Jira Issue ID
HARMONY-1671

## Description
Changes to assume internal load balancers use https instead of http.

## Local Test Steps
Test along with the ci-cd branch in sandbox. See instructions in ci-cd PR.
Sanity check no issues with local harmony which should be unaffected by the change.

## PR Acceptance Checklist
* [X] Acceptance criteria met
* [ ] Tests added/updated (if needed) and passing
* [ ] Documentation updated (if needed)

### CODE REVIEW

File: services/harmony/app/util/url.ts [hunk 1]
```diff
@@ -8,9 +8,7 @@ import * as url from 'url';
  */
 function _getProtocol(req): string {
   const host = req.get('host');
-  return (host.startsWith('localhost')
-    || host.startsWith('internal-harmony') // using an internal harmony load balancer
-    || host.startsWith('127.0.0.1')) ? 'http' : 'https';
+  return (host.startsWith('localhost') || host.startsWith('127.0.0.1')) ? 'http' : 'https';
```
Comment (indiejames) [inline services/harmony/app/util/url.ts:11]: do these work when Harmony is running in k8s?

**Comment by indiejames** [comment]
Code changes look fine to me. I will do a sandbox deployment tomorrow (assuming the VPN cooperates) and a harmony in a box deployment as well.

File: services/harmony/app/util/url.ts (hunk 1 above)
Comment (chris-durbin) [reply services/harmony/app/util/url.ts:11]: The only change to this line was removing internal-harmony so shouldn't be an issue. I verified I was able to successfully complete a request with harmony in a box after building all the images.

**Review by indiejames — APPROVED** [review]
Looks good. Chris demoed working requests using https in his sandbox.

**Pull request merged**